notebooks/intro.ipynb

```diff
@@ -1192,7 +1192,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.7.3"
```

Comment:
Does anyone know how to avoid this recurrent tiny change in the notebooks ?
This is kind of annoying :)

Reply:
thanks for pointing this out @aboucaud, I think at some point we can standarize the development environment for BTK (so we are all using the same version of python) and will make this issue go away. Other than that I'm not sure since I don't think nbstripout takes care of this right?

Reply:
To my knowledge nbstripout takes care of the output, not the metadata. But there is maybe an option.
Concerning the standardisation of the dev env, it usually does not go beyond specifying the minor version e.g. Python 3.7 so we would still have the issue from time to time.